en/includes/references/token-binding.md

```diff
@@ -0,0 +1,96 @@
+
+Token binding is a security mechanism in web protocols to establish a secure connection between an authentication token and the client device that holds it. The primary purpose of token binding is to prevent unauthorized token theft and replay attacks.
+
+!!! note
+    Learn how you can [register your application in {{product_name}}]({{base_path}}/guides/applications/) and [configure your application]({{base_path}}/references/app-settings/oidc-settings-for-app/)
+
+{{product_name}} supports the following token binding types for your OIDC applications.
+
+## Cookie
+
+Cookie token binding method binds the token to the cookie named `atbv` with `Secure` and `httpOnly` parameters. This method is supported with the **authorization code** grant type.
+
+## SSO-session
+
+SSO-session token binding method binds the access token to the login session. {{product_name}} issues a new access token for each new login and revoke the token upon logout. This method is supported with the **authorization_code** grant type.
+
+
+## Certificate
+
+Certificate token binding method binds the access token to the hash of the TLS certificate passed in the request. This method is supported with all grant types.
+
+## Device flow
+
+Device flow binding method binds the token to the `device_code` sent in the **device flow** grant type token call.
+
+
+## Client-request
+
+Back-channel grant types such as **token exchange** or **password**, cannot associate the user sessions with specific devices or instances. To address this issue, {{product_name}} introduced client request binding. The client-request binding type works with a parameter called `tokenBindingId`. This parameter allows developers to explicitly associate user sessions with specific devices or instances.
+
+Let's look at it in detail.
+
+The `tokenBindingId` parameter is used to reflect the user session in a particular device or instance. The behavior of the client-request binding type depends on the type of access token (JWT or Opaque) and the grant type that you configure for the application.
+
+With client-request token binding type enabled, for,
+
+- **JWT**
+
+    - If the request does not contain a `tokenBindingId`, {{product_name}} issues a new access token for every new token request with the same application, user and scope combination and revokes the previous tokens.
+
+    - If the request contains a `tokenBindingId`, {{product_name}} issues a new access token for each unique `tokenBindingId` value. Issuing a new token does not revoke tokens issued for other `tokenBindingId` values.
+
+- **Opaque**
+
+    - If the request does not contain a `tokenBindingId`, {{product_name}} issues the same access token for every new token request until it expires.
+
+    - If the request contains a `tokenBindingId`, {{product_name}} issues a new access token for each unique `tokenBindingId` value. Issuing a new token does not revoke tokens issued for other `tokenBindingId` values.
+
+### Try it out
```

Comment:
Will it be clumsy if we add try it out for all the binding types in the same doc?

Reply:
+1 to move this each binding type to new section once we have comprehensive doc for other types.

Reply:
We will separate the token types into their own docs when we have the content.